applications/ShapeOptimizationApplication/custom_utilities/geometry_utilities.h

```diff
@@ -196,6 +200,65 @@ class GeometryUtilities
     	KRATOS_CATCH("");
     }
 
+    // --------------------------------------------------------------------------
+    void ComputeProjectedDistancesToBoundingModelPart(
```

Comment:
```suggestion
    void ComputeDistancesToBoundingModelPart(
```

What about reducing the function name? This requires also a few adjustments further down the code. I see the meaning of "projected" but actually without further insight, the addition does not provide more information compared to the name without this addition. What do you think?

Reply:
done